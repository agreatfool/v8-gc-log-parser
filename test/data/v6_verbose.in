[43748:0x103000000] Memory allocator,   used:   7204 KB, available: 1459164 KB
[43748:0x103000000] New space,          used:    957 KB, available:     50 KB, committed:   2048 KB
[43748:0x103000000] Old space,          used:    888 KB, available:      0 KB, committed:    980 KB
[43748:0x103000000] Code space,         used:    242 KB, available:      0 KB, committed:   1024 KB
[43748:0x103000000] Map space,          used:     49 KB, available:      0 KB, committed:     80 KB
[43748:0x103000000] Large object space, used:      0 KB, available: 1458123 KB, committed:      0 KB
[43748:0x103000000] All spaces,         used:   2137 KB, available: 1458173 KB, committed:   4132 KB
[43748:0x103000000] External memory reported:      0 KB
[43748:0x103000000] Total time spent in GC  : 0.6 ms
[6785:0x103000000] Memory allocator,   used: 142148 KB, available: 1324220 KB
[6785:0x103000000] New space,          used:   7025 KB, available:   9094 KB, committed:  32768 KB
[6785:0x103000000] Old space,          used:  91744 KB, available:   8954 KB, committed: 102356 KB
[6785:0x103000000] Code space,         used:   1208 KB, available:      0 KB, committed:   2048 KB
[6785:0x103000000] Map space,          used:    265 KB, available:      0 KB, committed:   1104 KB
[6785:0x103000000] Large object space, used:    781 KB, available: 1323179 KB, committed:    800 KB
[6785:0x103000000] All spaces,         used: 101026 KB, available: 1341228 KB, committed: 139076 KB
[6785:0x103000000] External memory reported:     16 KB
[6785:0x103000000] Total time spent in GC  : 102.0 ms