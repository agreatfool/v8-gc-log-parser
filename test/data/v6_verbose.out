[
  {
    "pid": 43748,
    "isolate": "0x103000000",
    "type": "verbose",
    "data": {
      "memory_allocator": {
        "used": 7204,
        "available": 1459164
      },
      "new_space": {
        "used": 957,
        "available": 50,
        "committed": 2048
      },
      "old_space": {
        "used": 888,
        "available": 0,
        "committed": 980
      },
      "code_space": {
        "used": 242,
        "available": 0,
        "committed": 1024
      },
      "map_space": {
        "used": 49,
        "available": 0,
        "committed": 80
      },
      "large_object_space": {
        "used": 0,
        "available": 1458123,
        "committed": 0
      },
      "all_spaces": {
        "used": 2137,
        "available": 1458173,
        "committed": 4132
      },
      "external_memory_reported": 0,
      "total_time_spent_in_gc": 0.6
    }
  },
  {
    "pid": 6785,
    "isolate": "0x103000000",
    "type": "verbose",
    "data": {
      "memory_allocator": {
        "used": 142148,
        "available": 1324220
      },
      "new_space": {
        "used": 7025,
        "available": 9094,
        "committed": 32768
      },
      "old_space": {
        "used": 91744,
        "available": 8954,
        "committed": 102356
      },
      "code_space": {
        "used": 1208,
        "available": 0,
        "committed": 2048
      },
      "map_space": {
        "used": 265,
        "available": 0,
        "committed": 1104
      },
      "large_object_space": {
        "used": 781,
        "available": 1323179,
        "committed": 800
      },
      "all_spaces": {
        "used": 101026,
        "available": 1341228,
        "committed": 139076
      },
      "external_memory_reported": 16,
      "total_time_spent_in_gc": 102
    }
  }
]